fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerTest.java

```diff
@@ -367,4 +367,12 @@ protected <M extends Message> List<Tuple> getBoundariesList(List<M> records, fin
         boundaries.add(null);
         return boundaries;
     }
+
+    protected void snooze(int millis) {
+        try {
+            Thread.sleep(millis);
+        } catch (InterruptedException e) {
+            Thread.currentThread().interrupt();  //set the flag back to true
```

Comment:
I think you still want tho row a RuntimeException, right? If the thread is interrupted you don't want to continue to run the index build.
```suggestion
            Thread.currentThread().interrupt();  //set the flag back to true
            throw new RuntimeException(e);
```

Reply:
✅